raycaster/game/asset_loader.py

```diff
@@ -23,9 +23,12 @@ def __new__(cls):
             cls.WALL_TEXTURES_PATH = os.path.join(cls.ASSETS_PATH, "walls")
             cls.OBJECTS_SPRITES_PATH = os.path.join(cls.ASSETS_PATH, "objects")
             cls.STATIC_SPRITES_PATH = os.path.join(cls.OBJECTS_SPRITES_PATH, "static")
+            cls.HUB_TEXTURES_PATH = os.path.join(cls.ASSETS_PATH, "hub")
```

Comment:
shouldn't be `HUD_TEXTURES_PATH`?
```suggestion
            cls.HUD_TEXTURES_PATH = os.path.join(cls.ASSETS_PATH, "hud")
```